Cargo.toml

```diff
@@ -38,6 +38,7 @@ libc = "0.2.147"
 log = "0.4.20"
 rand = { version = "0.8.5", features = ["small_rng"] }
 slab = "0.4.9"
+socket2 = "0.4.7"
```

Comment:
Currently, this crate is only used on Linux. Placing it here would download it while building on Linux as well.

Reply:
Yes, that's intentional. I'm using the SockAddr -> SocketAddr functionality provided by socket2. Please advise if you want something different.

Reply:
Oh, I see. In that case, this looks good to me. Please proceed with this change.